app/plugin_api.go

```diff
@@ -262,8 +262,9 @@ func (api *PluginAPI) GetChannelByNameForTeamName(teamName, channelName string,
 	return api.app.GetChannelByNameForTeamName(channelName, teamName, includeDeleted)
 }
 
-func (api *PluginAPI) GetChannelsForTeamForUser(teamId, userId string, includeDeleted bool) (*model.ChannelList, *model.AppError) {
-	return api.app.GetChannelsForUser(teamId, userId, includeDeleted)
+func (api *PluginAPI) GetChannelsForTeamForUser(teamId, userId string, includeDeleted bool) ([]*model.Channel, *model.AppError) {
+	channels, err := api.app.GetChannelsForUser(teamId, userId, includeDeleted)
+	return *channels, err
```

Comment:
If `err` is not `nil`, `channels` likely will be and this will panic.

Reply:
I also missed this https://github.com/mattermost/mattermost-server/pull/9838. Will fix.